front/src/app/components/layouts/deal-list/view.tsx

```diff
@@ -55,29 +75,81 @@ export class DealListView extends React.PureComponent<IProps, any> {
                     record.endTime && now < record.endTime ? (
                         <div key="2">To: {end.format('D MMM YYYY H:mm')}</div>
                     ) : null,
-                    record.endTime && now < record.endTime ? (
-                        <div key="3">
-                            {Math.round(100 * (record.endTime - now) / 3600) /
-                                100}{' '}
-                            hour(s) left
-                        </div>
-                    ) : null,
                 ];
             },
         },
         {
-            className: 'sonm-cell-price sonm-deals__cell',
+            className: 'sonm-deals-list__cell-status',
+            dataIndex: 'status',
+            title: 'Type',
+            render: (status: string, record: IDeal) => {
+                const type =
+                    record.consumerID.toLowerCase() ===
+                    this.props.marketAccountAddress.toLowerCase()
+                        ? 'buy'
+                        : 'sell';
+                const cls = `sonm-deals-list__cell-status--${type}`;
+
+                return (
+                    <div className={cn('sonm-deals-list__cell-status', cls)}>
+                        {type}
+                    </div>
+                );
+            },
+        },
+        {
+            dataIndex: 'stats',
+            title: 'Stats',
+            render: (price: string, record: IDeal) => {
+                return (
+                    <div className="sonm-deals-list__cell-stats">
+                        <div className="sonm-deals-list__cell-stats__name">
+                            CPU Count
+                        </div>
+                        <div className="sonm-deals-list__cell-stats__value">
+                            {record.cpuCount}
+                        </div>
+                        <div className="sonm-deals-list__cell-stats__name">
+                            GPU ETH hashrate
+                        </div>
+                        <div className="sonm-deals-list__cell-stats__value">
```

Comment:
Did you see my long comments about BEM naming few days ago ?

It is incorrect  class name